### DISCUSSION TITLE
Something for Home Assistant // Grafana or similiar?

### DISCUSSION THREAD

**Discussion by 4920441** [mathoudebine/turing-smart-screen-python#126] [Ideas]
Hi,

I am searching for  a project which intergrated the turing display in Homeassistand or Grafana to display  some visualisation  about  Enegie consumption or generation.

Does anyone know if there is such a project already or  do I have to start one from scratch? :-)

Cheers

4920441

**Comment by mathoudebine** [comment]
Hello @4920441
I don't think the screen has been integrated to Home Assistant yet
You can search Github for "turing screen" in case someone has made something similar.
I'm thinking e.g. https://github.com/ferni7/turing-smart-screen-custom

Otherwise if you're interested to start this from scratch, here is the documentation to control the display from your own code: https://github.com/mathoudebine/turing-smart-screen-python/wiki/Control-screen-from-your-own-code

Good luck!
